.eslintrc

```diff
@@ -4,20 +4,30 @@
     "strict": 0,
     "no-underscore-dangle": 0,
     "no-unused-vars": 0,
-    "curly": 0,
     "no-multi-spaces": 0,
     "key-spacing": 0,
     "no-return-assign": 0,
-    "consistent-return": 0,
+    "consistent-return": 1,
     "no-shadow": 0,
-    "no-comma-dangle": 0,
-    "no-use-before-define": 0,
-    "no-empty": 0,
-    "new-parens": 0,
-    "no-cond-assign": 0,
-    "quotes": "single"
+    "comma-dangle": [1, "never"],
+    "no-use-before-define": 1,
+    "new-parens": 1,
+    "no-cond-assign": 1,
+    "quotes": [1, "single"],
+    "eqeqeq": [2, "smart"],
+    "wrap-iife": [2, "outside"],
+    "indent": [2, 2],
+    "brace-style": [2, "1tbs"],
+    "space-after-keywords": 2,
+    "space-before-function-parentheses": [2, {
+      "anonymous": "always",
+      "named": "never"
+    }],
+    "space-infix-ops": 2,
+    "spaced-line-comment": [1, "always", {exceptions:["-","+"]}]
```

Comment:
Good catch.  I yanked this straight from the docs (http://eslint.org/docs/rules/spaced-line-comment.html) so I also created https://github.com/eslint/eslint/pull/2019 to update the docs.